src/reachy2_sdk/orbita/orbita.py

```diff
@@ -50,10 +53,10 @@ def __init__(
 
         self._compliant: bool
 
-        self._state: Dict[str, bool] = {}
-        self._register_needing_sync: List[str] = []
         self._joints: Dict[str, Any] = {}
+        self._reversed_joints: Dict[Any, str] = {}
         self._motors: Dict[str, OrbitaMotor] = {}
+        self._waiting_goal_positions: Dict[str, float] = {}
```

Comment:
I did not understand *waiting* directly here. But maybe it is just me. What do you think about outgoing instead?

Reply:
renamed